test/renderer/components/ui/MemberSelect.test.tsx

```
@@ -0,0 +1,174 @@
+          ]}
+          value="team-lead"
+          onChange={onChange}
+          searchPlaceholder="Search log sources..."
+          emptyMessage="No log sources found."
+          ariaLabel="Log source"
+          getMemberLabel={(candidate) =>
+            candidate.name === 'team-lead'
+              ? 'Lead'
+              : candidate.removedAt
+                ? `${candidate.name} (removed)`
+                : candidate.name
```

ESLint

[error] Extract this nested ternary operation into an independent statement.

(sonarjs/no-nested-conditional)